### PR TITLE
Feature/ DAO-808

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

- add eslint
- clean subgraph testing

Task: [ID](https://aragonassociation.atlassian.net/browse/DAO-808)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/subgraph/src/dao/utils.ts [hunk 1]
```diff
@@ -23,7 +22,9 @@ class WithdrawParams {
   token: Address = Address.fromString(ADDRESS_ZERO);
   to: Address = Address.fromString(ADDRESS_ZERO);
   amount: BigInt = BigInt.zero();
+  /* eslint-disable */
```
Comment (mathewmeconry) [inline packages/subgraph/src/dao/utils.ts:25]: you can disable a certain rule only for the next line with `// eslint-disable-next-line @typescript-eslint/no-empty-interface`
much cleaner and doesn't have the chance that you can forget to enable it again

**Review by mathewmeconry — APPROVED** [review]

File: packages/subgraph/src/utils/erc165.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import {ethereum, Bytes} from '@graphprotocol/graph-ts';
 export function supportsInterface(
   contract: ethereum.SmartContract,
   interfaceId: string,
-  expected: boolean = true
+  expected = true
```
Comment (Michael-A-Heuer) [inline packages/subgraph/src/utils/erc165.ts:6]: Is this an improvement?

**Review by Michael-A-Heuer — DISMISSED** [review]

File: packages/subgraph/src/dao/utils.ts (hunk 1 above)
Comment (Rekard0) [reply packages/subgraph/src/dao/utils.ts:25]: thanks, amended

File: packages/subgraph/src/utils/erc165.ts (hunk 2 above)
Comment (Rekard0) [reply packages/subgraph/src/utils/erc165.ts:6]: yes, if a default value is assigned it get the type from it
Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types

**Review by mathewmeconry — APPROVED** [review]

**Review by Michael-A-Heuer — APPROVED** [review]

**Pull request merged**